yt_dlp/extractor/ceskatelevize.py

```diff
@@ -106,67 +127,94 @@ def _real_extract(self, url):
         if playlist_description:
             playlist_description = playlist_description.replace('\xa0', ' ')
 
-        type_ = 'IDEC'
+        type_ = 'episode'
+        is_live = False
         if re.search(r'(^/porady|/zive)/', parsed_url.path):
             next_data = self._search_nextjs_data(webpage, playlist_id)
             if '/zive/' in parsed_url.path:
                 idec = traverse_obj(next_data, ('props', 'pageProps', 'data', 'liveBroadcast', 'current', 'idec'), get_all=False)
+                sidp = traverse_obj(next_data, ('props', 'pageProps', 'data', 'liveBroadcast', 'current', 'showId'), get_all=False)
```

Comment:
Apparently `sidp` now answers to its name, but in case there is still some use of `showId`:
 ```suggestion
                current_bcast = traverse_obj(next_data, ('props', 'pageProps', 'data', 'liveBroadcast', 'current'))
                idec, sidp = (traverse_obj(current_bcast, *path)
                              for path in (('idec',), ('sidp', 'showId')))
```